Makefile

```diff
@@ -68,14 +68,23 @@
 #         place.
 #
 
+# The PREFIX variable is used to set up POPLOG_HOME_DIR and EXEC_DIR (and 
+# nowhere else, please). It is provided in order to fit in with the conventions 
+# of Makefiles. 
+DESTDIR?=/
```

Comment:
I'd just `DESTDIR?=` so that if users invoke `DESTDIR=/tmp make install` then it will still work. The current implementation will instead install to `/tmpusr/local`